src/kitovu/sync/settings.py

```diff
@@ -84,7 +84,10 @@ def from_yaml_stream(cls, stream: typing.IO,
             validator = utils.SchemaValidator()
 
         # FIXME handle OSError and UnicodeDecodeError
-        data = yaml.load(stream)
+        try:
+            data = yaml.load(stream)
+        except yaml.scanner.ScannerError as error:
+            raise utils.UsageError(f"Invalid Configuration:\n{str(error)}")
```

Comment:
Formatting things into strings automatically stringifies them, so no need to add a `str()`

Reply:
Oh yeah, right. I've removed it and added a test